src/main/osd/osd_elements.c

```diff
@@ -1858,6 +1863,8 @@ void osdElementsInit(bool backgroundLayerFlag)
 {
     backgroundLayerSupported = backgroundLayerFlag;
     activeOsdElementCount = 0;
+    pt1FilterInit(&batteryEfficiencyFilt, 0.0f);
```

Comment:
`0.0f` is not good `gain`.  `gain` should be computed using `pt1FilterGain` based on

- `dT` sampling period in seconds (float number).
- `f_cut` - cutoff frequency in Hz


You probably don't need to call `pt1FilterUpdateCutoff` as it is the same `k` number as gain in Init.